spark_client/services.py

```diff
@@ -654,7 +660,7 @@ def set_primary(self, account_id: str) -> str:
 
         if any([account.primary for account in self.all()]):
             self.logger.info("Switching primary account")
-            for account_id, account in self.cache.items():
+            for account_id_i, account in self.cache.items():
```

Comment:
If not used, we could also just use the placeholder "_" (it is sort of a convetion) or rather do "self.cache.values()" instead